README.md

```diff
@@ -48,21 +48,31 @@ Note that Vroom requires a version of vim built with the `+clientserver`
 option (run `vim --version` to check).  See `:help clientserver` for
 additional requirements.
 
-The easiest way to install vroom is to cd into the vroom directory and run
+If you're on Ubuntu or Debian, you can install
+[release packages](https://github.com/google/vroom/releases) from GitHub.
+
+Otherwise, the easiest way to install vroom is to clone the vroom repository
+from Github, cd into the vroom directory, and run
 ```sh
 python3 setup.py build && sudo python3 setup.py install
```

Comment:
I think I wanted to and someone suggested not to lead with that since it has the extra caveat of needing ~/.local/bin on your `$PATH`. We should probably create an INSTALLING doc with more of these options and alternatives and link it from here. I'll leave it up to you if you want to suggest some instructions about `install --user` to insert here.